File: Android/app/src/main/java/com/example/liquorapplication/Order.kt
package com.example.liquorapplication

data class Order(
    val id: String,
    val docType: String = "Order",
    val orderId: Int,
    val storeId: String,
    val orderDate: Long,
    val orderStatus: String, // "Submitted", "Received"
    val productId: Int,
    val sku: String,
    val unit: String,
    val orderQty: Int
)

File: iOS/LiquorApp/Order.swift
import Foundation

struct Order: Identifiable, Codable {
    let id: String
    let docType: String
    let orderId: Int
    let storeId: String
    let orderDate: Int64
    let orderStatus: String // "Submitted", "Received"
    let productId: Int
    let sku: String
    let unit: String
    let orderQty: Int
    
    var orderDateFormatted: String {
        let date = Date(timeIntervalSince1970: TimeInterval(orderDate / 1000))
        let formatter = DateFormatter()
        formatter.dateStyle = .medium
        formatter.timeStyle = .short
        return formatter.string(from: date)
    }
}

